packages/trader/src/App/Components/Elements/EmptyPortfolioMessage/__tests__/empty-portfolio-message.spec.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import EmptyPortfolioMessage from '../empty-portfolio-message';
+
+describe('EmptyPortfolioMessage', () => {
+    const empty_portfolio_message =
+        'You have no open positions for this asset. To view other open positions, click Go to Reports';
+    const error_text = 'Error text';
+
+    it('should render EmptyPortfolioMessage component', () => {
+        const { container } = render(<EmptyPortfolioMessage />);
+
+        expect(container).toBeInTheDocument();
```

Comment:
same as in here https://github.com/binary-com/deriv-app/pull/8399/files#r1182215163